src/_guides/language/language-tour.md

```diff
@@ -690,6 +700,17 @@ var nobleGases = {
 };
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  If you're wondering whether these map literals are [strong
+  mode compliant](/guides/language/sound-dart), they are. Thanks to
+  [type inference](/guides/language/sound-dart#type-inference),
+  the analyzer infers that the first map, for example,
+  contains keys and values of type `String`. If you try to add
```

Comment:
Similarly here,

...infers that `gifts`, for example, has the type `Map<String, String>`.